2/js/godinhojoao/tabuada.js

```diff
@@ -0,0 +1,21 @@
+function multiplicationTable(number) {
+    number = parseInt(number);
+
+    for (let i = 1; i <= number; i++) {
+        console.log('===========================');
+        console.log(`A tabuada do numero ${i} é:`);
+        console.log(`${i} * 1 = ${i * 1}`);
+        console.log(`${i} * 2 = ${i * 2}`);
+        console.log(`${i} * 3 = ${i * 3}`);
+        console.log(`${i} * 4 = ${i * 4}`);
+        console.log(`${i} * 5 = ${i * 5}`);
+        console.log(`${i} * 6 = ${i * 6}`);
+        console.log(`${i} * 7 = ${i * 7}`);
+        console.log(`${i} * 8 = ${i * 8}`);
+        console.log(`${i} * 9 = ${i * 9}`);
+        console.log(`${i} * 10 = ${i * 10}`);
```

Comment:
Você pode alterar isso para um outro **for** iterando de 1 a 10
```js
for (let j = 1; j <= 10; j++) {
  console.log(`${i} x ${j} = ${i * j}`);
}

```

Reply:
Obrigado, irei fazer isso agora !!